doc/library/config.txt

```diff
@@ -370,6 +370,33 @@ import theano and print the config variable, as in:
     `amdlibm <http://developer.amd.com/cpu/libraries/libm/>`__
     library, which is faster than the standard libm.
 
+.. attribute:: lib.cnmem
+
+    Float value: >= 0
```

Comment:
I'm not sure I like that this is a float value.  I would like it better to have two separate config vars to enable/disable and to set the allocated memory.

But I don't feel too strongly about this.